components/event-sources/Gopkg.toml

```diff
@@ -0,0 +1,36 @@
+required = [
+  "k8s.io/code-generator/cmd/deepcopy-gen",
+  "k8s.io/code-generator/cmd/client-gen",
+  "k8s.io/code-generator/cmd/lister-gen",
+  "k8s.io/code-generator/cmd/informer-gen",
+  "knative.dev/pkg/codegen/cmd/injection-gen",
+]
+
+[prune]
+  go-tests = true
+  unused-packages = true
+
+# Direct dependencies
+[[constraint]]
+  name = "knative.dev/pkg"
+  branch = "release-0.10"
+[[constraint]]
+  name = "knative.dev/serving"
+  version = "v0.10.0"
```

Comment:
currently we ship with 0.8.1
this means we should update to 0.10.0

Reply:
Not necessarily, this affects only the sources controller itself, not the whole event-bus. The CRDs in this repo are inside a Kyma API group, not *.knative.dev.